app/services/katello/pulp3/repository/file.rb

```diff
@@ -8,12 +8,14 @@ def copy_content_for_source(source_repository, _options = {})
           copy_units_by_href(source_repository.files.pluck(:pulp_id))
         end
 
-        def distribution_options(path)
-          {
+        def distribution_options(path, unprotected = true)
+          options = {
             base_path: path,
             publication: repo.publication_href,
             name: "#{generate_backend_object_name}"
           }
+          options[:content_guard] = Katello::Pulp3::ContentGuard.first.pulp_href unless unprotected
+          options
```

Comment:
would it make sense to have a 'common_distribution_options' method to handle the cert guard piece?